### ISSUE TITLE
Appodeal SDK Manager - dependency update/remove does not work (v3.0.0)

### ISSUE THREAD

**Issue by Bonbeart** [appodeal/appodeal-unity-plugin-upm#7]
## Step 1: Environment

    Computer OS: Windows 10
    Unity Editor Version: 2020.3.38f1
    Appodeal Unity Plugin Version: 3.0.0
    Bug Repro Rate: 100%

## Step 2: Describe the bug
In plugin version 3.0.0 - dependency update/remove does not work.

## Step 3: How to reproduce
1. Install Appodeal v3.0.0 by UPM 
2. Apply this fix to open **Appodeal SDK Manager** (info from Appodeal tech support)
Open file: `Appodeal/Editor/SDKManagerWindow/AppodealDependencyUtils.cs` and change method `GetConfigName()` to:
```
public static string GetConfigName(string value) {
    string configName = value.Replace("\\", "/"); 
    configName = configName.Replace($"{AppodealEditorConstants.PluginPath}/{AppodealEditorConstants.DependenciesPath}/", String.Empty);
    return configName.Replace("Dependencies.xml", String.Empty);
}
```

## Step 4: Expected behavior
Dependencies are updated and deleted without any problems.

## Step 5: Screenshots
![Unity_tIq4zLyB8V](https://user-images.githubusercontent.com/39597999/200064531-ee42e527-90dd-4e18-b5c9-4716bf5b94ad.gif)

## Step 6: Additional context

Now I do not understand at all - what-I have to do in this stalemate situation.
v3.0.1 - not working yet
v3.0.0 - broken
v2.+ - not supported by GooglePlay

You do not have any working version of the SDK, which I could release to GooglePlay!

**Comment by DmitriiFeshchenko** [comment]
@Bonbeart, as I can see, per our agreement at #5, you have an active intercom chat with our support team. Please, give them some time to answer your question before creating any further issues on GitHub.

I'm closing this issue to avoid work duplication.

**Comment by Bonbeart** [comment]
Is talking to tech support - somehow cancels the existence of this problem in your SDK, which I reported.

If you mean about the application of [this solution](https://wiki.appodeal.com/en/faq-troubleshooting/troubleshooting/unity-common-issues/adapters-removal) - it was also applied, and no changes...

The thing is, I currently need a working solution of your SDK, and it does not exist... 
Support, as always, promises "next week" - as it promised a year ago, nothing has changed since then...

**Comment by DmitriiFeshchenko** [comment]
@Bonbeart, this is a known issue that was already fixed in v3.0.1.

It's really sad to hear you experienced delays with our support team. I did ask them to help you asap.

The simplest solution, as I told you in #5, is to manually change the content of the files. Just send the dependency names to our support and ask for the latest versions of the corresponding XML files.

**Comment by Bonbeart** [comment]
Thanks!

I finally was able to fix it - using some changes from the 3.0.1 branch to the current 3.0.0.
